server/server.js

```diff
@@ -73,6 +80,44 @@ app.post('/reviews', (req, res) => {
   );
 });
 
+let randomizeReviews = function() {
```

Comment:
I'm a little lost about some of the structure of this function.  It appears that min is always 0 and max is always 14, is there a reason for the .floor and .ceil declarations at the top? It doesn't seem that min or max is reassigned at any point, so you may be able to do away with those top two lines and have `Math.floor(Math.random() * 15);` on line 89

Reply:
Also, `console.log()`'s